src/main/resources/db/migration/claimstore/V2018_09_20_1526__Add_money_received_column.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE claim
+  ADD COLUMN moneyReceived DATE;
```

Comment:
we've underscore pattern in DB e.g. money_received_on like other column attributes